waspc/src/Wasp/Util.hs

```diff
@@ -190,3 +192,9 @@ hexFromString = Hex
 
 hexToString :: Hex -> String
 hexToString (Hex s) = s
+
+allValues :: (Bounded a, Enum a) => [a]
+allValues = [minBound ..]
```

Comment:
Read more about it here: https://stackoverflow.com/questions/4299319/getting-a-list-of-all-possible-data-type-values-in-haskell

Reply:
Yup, this is cool! I wonder if it makes sense to make a util function for this, or just inline it? I guess we can have this function! Maybe call it `allValuesOfAType`? And then even fancier, you could call it with `allValuesOfAType @TokenKeyword` if you use TypeApplications extension.